src/components/KitchenNotes/index.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState, useEffect } from 'react'
+import { db } from '../../firebase'
+import { v4 as uuidv4 } from 'uuid'
+import { toast, Slide } from 'react-toastify'
+import 'react-toastify/dist/ReactToastify.css'
+import Masonry from 'react-masonry-css'
+import './Note.css'
+
+// needed for toasts
+toast.configure()
+
+function KitchenNotes(){
+
+    const [ incoming, setIncoming ] = useState([])
+
+    const getOrders = () =>{
+        return db
+        .collection('orders')
+        .where("status", "==", "in progress")
+        .orderBy('date', 'desc')
+        .onSnapshot((querySnapshot) => {
+            const docs = []
+            querySnapshot.forEach((doc) => {
+                docs.push({...doc.data(), id: doc.id})
+            })
+            setIncoming(docs)
+        })
+    } 
+
+    useEffect(() => {
+        let unsubscribe = getOrders()
+
+        return () => {
+            unsubscribe()
+        }
+      }, []);
+
+    const updateOrder = (id) => {
+        db    
+        .collection('orders')
+        .doc(id)
+        .update({
+            status: "done"
+        })
+        .then(() => {
+            toast.success('Un meserx será notificado', {
+                className: "rounder-edges",
+                position: "top-center",
+                autoClose: 4000,
+                hideProgressBar: true,
+                closeOnClick: true,
+                pauseOnHover: true,
+                draggable: true,
+                progress: undefined,
+                transition: Slide
+                });
+        })
+        .catch(() =>{
+            toast.error('Ocurrió un error, inténtalo de nuevo en un momento', {
+                className: "rounder-edges",
+                position: "top-center",
+                autoClose: 4000,
+                hideProgressBar: true,
+                closeOnClick: true,
+                pauseOnHover: true,
+                draggable: true,
+                progress: undefined,
+                transition: Slide
```

Comment:
Tanto el success and error toast comparten los mismos settings. Sería una buena idea tener estos settings a nivel de `toast.configure()`, de esta forma evitamos repetir los settings una y otra vez y centralizamos los settings para todos los notificationes toast